src/main/resources/tmp.json

```diff
@@ -0,0 +1,7 @@
+{
+  "firstName": "test_1f48359511e5",
+  "lastName": "test_27d49905ee0d",
+  "pesel": "test_85b996057a68",
+  "gender": "FEMALE",
+  "dateOfBirth": "2030-04-01"
+}
```

Comment:
to jest śmieć i nie powinien być dołączany do repozytorium

Reply:
usunięte